### DISCUSSION TITLE
Aside from the 🦒, what is the only other living member of the giraffe family?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#687] [Galaxy Trivia] [answered]
**Bonus:** Where do they live?

**Comment by DevElkami** [comment]
Known as the “forest giraffe,” the okapi looks more like a cross between a deer and a zebra. Nevertheless, it's the giraffe's only living relative.

The Okapi is an endemic species of the Democratic Republic of the Congo and is the national conservation symbol of the country.

**Comment by peterbaccaro** [comment]
The **okapi** and the giraffe are the only living members of the family Giraffidae.
Wild okapi live exclusively in the **Ituri Rainforest** in northwest Democratic Republic of the Congo in central Africa.

**Comment by SANKARAMDAS** [comment]
Although the okapi has striped markings reminiscent of zebras, it is most closely related to the giraffe. Known as the “forest giraffe,” the okapi looks more like a cross between a deer and a zebra. Nevertheless, it's the giraffe's only living relative.
The okapi and the giraffe are the only living members of the family Giraffidae.

**Comment by thuongtruong1009** [comment, marked as answer]
The only other living member of the giraffe family is the okapi (Okapia johnstoni).

Okapis are found in the dense rainforests of central Africa, specifically in the Democratic Republic of the Congo (DRC). They have a similar body shape to giraffes, with a long neck and legs, but they have a much shorter neck and a dark brown coat with white stripes on their legs.

Okapis are very reclusive animals and are rarely seen in the wild. They are listed as Endangered on the IUCN Red List due to habitat loss and hunting, and there are estimated to be only around 10,000 individuals remaining in the wild.
